src/software/ai/hl/stp/play/free_kick_play.cpp

```diff
@@ -74,12 +74,12 @@ void FreeKickPlay::getNextTactics(TacticCoroutine::push_type &yield, const World
     if (world.ball().position().x() > -1)
     {
         double y_offset =
-            -std::copysign(world.field().xLength() / 2, world.ball().position().y());
+            -std::copysign(world.field().yLength() / 2, world.ball().position().y());
```

Comment:
What does y_offset mean? Can we rename to something more meaningful?

Reply:
renamed, lmk if that's better.